cmd/upgrade/upgrade.go

```diff
@@ -102,7 +102,7 @@ func upgrade(u *updater.Updater) (string, error) {
 		}
 		return fmt.Sprintf("Successfully updated to version %s", latestVersion), nil
 	}
-	return "", u.Rollback()
+	return "", nil
```

Comment:
Why do we need this change ? 
```
# if this condition failed then it is failed upgrade and we want to roll back.
if updateStatus == updater.Updated {
}
```

Reply:
I have no idea why this comment is there, but this is not the case.
If the update fails then go-rocket-update will call `Rollback()` itself if needed.
And if the update fails then it will return an error... here we call rollback when the call to `Update()` returns no error and when status is not `Updated`.
This means if the status is `UpToDate` you are calling `Rollback()` (why?)

Reply:
got it thanks